source/appModules/securecrt.py

```diff
@@ -1,23 +1,28 @@
 #appModules/securecrt.py
+# Modified to include a regular expression so that all Secure CRT versions are supported past, present, and future.
+# Modification of original securecrt.py by Noel Romey 9/2016
 #A part of NonVisual Desktop Access (NVDA)
 #This file is covered by the GNU General Public License.
 #See the file COPYING for more details.
-#Copyright (C) 2010-2012 NV Access Limited

```

Comment:
I'd prefer the additional comments to be removed; just add your name to the copyright. Comments to describe what something does are great, but we don't need to comment when or how something was modified, as we can use the commit history for this.